### ISSUE TITLE
[MSFT-36] [Login] user-scalable=no on <meta> tag disables zooming on mobile devices

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2758]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2. Open the source

```
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
```

**Expected Result**
Ensures <meta name="viewport"> does not disable text scaling and zooming

**Actual Result**
user-scalable=no on <meta> tag disables zooming on mobile devices

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.4 Resize Text
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F20

**Comment by mishacucicea** [comment]
Will be fixed by #2772.

**Comment by Oanh-and-only-Oanh** [comment]
closing because it's a duplicate bug
